### PR TITLE
Update Profiler and Memory Profiler

### PR DESCRIPTION
**Pull request by mijin85cho**
Signed-off-by: Mijin, Cho <mijin85.cho@samsung.com>

### Change Description ###

Update Profiler user manual and Memory profiler user manual by request of seok.oh.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/442

**Comment by mijin85cho** [comment]
Hello, @annie-samsung 
I want to check the review schedule of this PR with you.
Is it possible to finish the review till 10.30?

If so, we will process the review.

If not, as we have to merge this PR till 10.31, 
I hope to merge this PR just as it is for now,
and do review and update later, after 10.31.

Would you please check?

File: docs/application/vstools/index.md [hunk 1]
```diff
@@ -34,10 +34,14 @@ Visual Studio Tools for Tizen provides Tizen-specific tools to improve your prod
...
   The Log Viewer allows you to check the logs of your application based on the logging methods you have inserted into your code. It helps you to debug your application by capturing all the events logged by the platform and your application.
 
-- [Profiler](tools/profiler.md)
+- [Profiler](tools/profiler-user-manual.md)
 
   The Profiler allows you to analyze the performance of your application. You can record, manage, and view profiling sessions.
 
+- [Memory Profiler](tools/memory-profiler-user-manual.md)
+
+  This is a utility which collects memory profiling data and save them to a file, this part of the Memory Profiler runs on the target side. Also it analyzes the resulting data, the command-line one and the GUI-based one, these analyzing tools run on the host side.
```
Comment (divyk) [inline docs/application/vstools/index.md]: Please review the rephrase and check whether the meaning of the content is not changed:

Memory Profiler is an utility, which collects memory profiling data and saves them to a file. Usually, the Memory Profiler runs on the target side. In addition, it analyzes the resulting data, from both command-line and GUI-based. These analyzing tools runs on the host side.

**Comment by divyk** [comment]
I will review and submit the Profiler user manual before end of day.

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  - Additional session's information is displayed in **Properties** window (usually it is placed below **Session Explorer** window). The window contains information about specific session's parameters that had been used to configure the profiler for that session. Annotation field can be edited in this window.
+    
+      ![Session Properties](media/session_properties.png)
+
+  - To open session double click on the session's entry.
+  - To manage session click `Right Mouse Button` on the session's entry.
+      
+    ![Session's entry Context menu](media/mouse_right.png)
+
+    The next possible operations with session entry are available:
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Following are the next possible operations with session entry:

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+    - Open Profile Session
+    - Open Memory Profile Session (availability depends on the session's parameters)
+    - Edit Annotation
+    - Delete
+
+## Analyzing session data
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Analyzing Session Data

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+  Depends on the chosen Profiling preset there are available two types of the session data's viewers:
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Depending on the Profiling preset, following are the types of the session data's viewers:

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+  - Profile Session viewer - if **CPU Samplings** or **Complex** preset had been chosen.
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Profile Session viewer - if you choose **CPU Samplings** or **Complex** preset.

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+  - Memory Profile Session viewer - if **Memory Allocation** or **Complex** preset had been chosen.
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Memory Profile Session viewer - if if you choose **Memory Allocation** or **Complex** preset.

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+
+  If **Complex** preset have been chosen both types of sessions will be available.
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: > Note:
>
> If you choose **Complex** preset, both types of sessions will be available.

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+
+### Profile Session viewer
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Profile Session Viewer

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+
+  ![Profile Session](media/profile_session_viewer.png)
+
+  The window wll be automatically shown during profiling or live profiling.
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: This line can be placed above the figure:

The following window is automatically shown during profiling or live profiling:

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 10]
```diff
@@ -0,0 +1,263 @@
...
+
+  To open it manually select in the menu **Tools > Tizen > Profiler > Profiling Progress**.
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: To open it manually, from the menu, select **Tools > Tizen > Profiler > Profiling Progress**.

**Comment by mijin85cho** [comment]
Discussion with @winstone77

I'll merge this PR  as it is for now,
and merge winstone77 will raise a new PR for the update later.
Thanks for the review @divyk.
We'll come back with the comments from you later.

File: docs/application/vstools/tools/profiler-user-manual.md [hunk 11]
```diff
@@ -0,0 +1,263 @@
...
+
+  The viewer has the next panels:
```
Comment (divyk) [inline docs/application/vstools/tools/profiler-user-manual.md]: Please review the rephrase and check whether the meaning of the content is not changed:

Following are the available panels:

**Comment by divyk** [comment]
Ok, @mijin85cho, I will work in this PR later when it is available. Thanks.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**